src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs

```diff
@@ -61,20 +61,20 @@ private string TenantId
             }
         }
 
-        public override void LogFunctionTraceEvent(LogLevel level, string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string exceptionType, string exceptionMessage, string functionInvocationId, string hostInstanceId, string activityId)
+        public override void LogFunctionTraceEvent(LogLevel level, string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string exceptionType, string exceptionMessage, string functionInvocationId, string hostInstanceId, string activityId, string runtimeSiteName)
         {
             string eventTimestamp = DateTime.UtcNow.ToString(EventTimestampFormat);
             string hostVersion = ScriptHost.Version;
             FunctionsSystemLogsEventSource.Instance.SetActivityId(activityId);
 
-            _writeEvent($"{ScriptConstants.LinuxLogEventStreamName} {(int)ToEventLevel(level)},{subscriptionId},{appName},{functionName},{eventName},{source},{NormalizeString(details)},{NormalizeString(summary)},{hostVersion},{eventTimestamp},{exceptionType},{NormalizeString(exceptionMessage)},{functionInvocationId},{hostInstanceId},{activityId},{_containerName},{StampName},{TenantId}");
+            _writeEvent($"{ScriptConstants.LinuxLogEventStreamName} {(int)ToEventLevel(level)},{subscriptionId},{appName},{functionName},{eventName},{source},{NormalizeString(details)},{NormalizeString(summary)},{hostVersion},{eventTimestamp},{exceptionType},{NormalizeString(exceptionMessage)},{functionInvocationId},{hostInstanceId},{activityId},{runtimeSiteName},{_containerName},{StampName},{TenantId}");
         }
 
-        public override void LogFunctionMetricEvent(string subscriptionId, string appName, string functionName, string eventName, long average, long minimum, long maximum, long count, DateTime eventTimestamp, string data)
+        public override void LogFunctionMetricEvent(string subscriptionId, string appName, string functionName, string eventName, long average, long minimum, long maximum, long count, DateTime eventTimestamp, string data, string runtimeSiteName)
         {
             string hostVersion = ScriptHost.Version;
 
```

Comment:
Linux event generators can't handle dynamic event manifest changes like etw can. Can you remove the event changes for LinuxAppServiceEventGenerator and LinuxContainerEventGenerator? This will otherwise break functions logging on linux. We can discuss offline for how to handle this change for linux.

Reply:
@divyagandhii - It would be great if you can fill me in on details as well as I would like understand linux logging pipeline

Reply:
@pragnagopa ya sure